regen/warnings.pl

```diff
@@ -884,6 +884,9 @@ =head1 SYNOPSIS
     use warnings;
     no warnings;
 
+    # Standard warnings are enabled by use v5.35 or above
+    use v5.35;
```

Comment:
Can someone explain to me the purpose of this script and why what seems to be a functional change when the _ask_ here was for documentation?

Reply:
This is the documentation for warnings.pm.